mlflow/resource_experiment_test.go

```diff
@@ -0,0 +1,115 @@
+package mlflow
+
+import (
+	"testing"
+
+	"github.com/databrickslabs/terraform-provider-databricks/qa"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestMLFlowExperimentCreate(t *testing.T) {
+	d, err := qa.ResourceFixture{
+		Fixtures: []qa.HTTPFixture{
+			{
+				Method:   "POST",
+				Resource: "/api/2.0/mlflow/experiments/create",
+				ExpectedRequest: MLFLowExperimentAPI{
+					Name: "xyz",
+					Tags: []Tag{
+						{"key1", "value1"},
+						{"key2", "value2"},
+					},
+				},
+				Response: MLFLowExperimentAPI{
+					ExperimentId: "123456790123456",
+					Name:         "xyz",
+					Tags: []Tag{
+						{"key1", "value1"},
+						{"key2", "value2"},
+					},
+				},
+			},
+			{
+				Method:   "GET",
+				Resource: "/api/2.0/mlflow/experiments/get?experiment_id=123456790123456",
+				Response: MLFLowExperimentsAPI{
+					MLFLowExperimentAPI{
+						ExperimentId: "123456790123456",
+						Name:         "xyz",
+						Tags: []Tag{
+							{"key1", "value1"},
+							{"key2", "value2"},
+						},
+					},
+				},
+			},
+		},
+		Resource: ResourceMLFlowExperiment(),
+		Create:   true,
+		HCL: `
+		name = "xyz"
+		tags {
+				key = "key1"
+				value = "value1"
+		    }
+		tags {
+				key = "key2"
+				value = "value2"
+			  }
+		`,
+	}.Apply(t)
+
+	assert.NoError(t, err, err)
+	assert.Equal(t, "123456790123456", d.Id(), "Resource ID should not be empty")
```

Comment:
If you used a variable as suggested in previous comment, you can assert for experiment.ExperimentId here, instead of the "123..." string which can change at any time.